src/features/todos.ts

```diff
@@ -1,8 +1,33 @@
-import { createSlice } from '@reduxjs/toolkit';
+/* eslint-disable @typescript-eslint/default-param-last */
 import { Todo } from '../types/Todo';
 
-export const todosSlice = createSlice({
-  name: 'todos',
-  initialState: [] as Todo[],
-  reducers: {},
+type SetTodosAction = {
+  type: 'todos/SET';
+  payload: Todo[];
+};
+
+const setTodos = (todos: Todo[]): SetTodosAction => ({
+  type: 'todos/SET',
+  payload: todos,
 });
+
+export const actions = { setTodos };
+
+type Action = SetTodosAction;
+
+const todosReducer = (state: Todo[] = [], action: Action): Todo[] => {
+  switch (action.type) {
+    case 'todos/SET':
+      return action.payload;
+
+    default:
+      return state;
+  }
+};
+
+export default todosReducer;
+
+// Задание: Распределение уникальных комбинаций
+// Напишите функцию, которая принимает массив чисел и целевое число target,
+// и возвращает все уникальные комбинации чисел, где сумма каждой комбинации равна target.
+// При этом каждое число может быть использовано ровно один раз.
```

Comment:
This comment about a task involving unique combinations of numbers seems unrelated to the todos reducer. Consider removing it if it's not relevant to the current file's purpose.